.github/workflows/cron_docker_build.yml

```diff
@@ -19,6 +19,7 @@ jobs:
           - 7.4
           - 8.0
           - 8.1
+          - 8.2
```

Comment:
PrestaShop 8.2 is not compatible with PHP 8.2 so I don't think we should add this version on this branch